phpmd 2.6.0 (new formula)

JoyceBabu opened the pull request:
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

javian commented on Formula/phpmd.rb:
```diff
@@ -0,0 +1,28 @@
+
+  bottle :unneeded
+
+  depends_on "php"
```
@SMillerDev since Macos ships with php and phpmd only requires php 5.3.9 would it not be possible for us to skip the dependency here ?

SMillerDev replied:
Good catch, I didn't think of that.

javian replied:
@JoyceBabu can you please remove the php dependency since what ships with MacOS is good enough to support this ?

JoyceBabu commented:
So that explains why the `composer` package did not have the `php` dependency. I think the `php` dependency can be removed from `phplint` package too, as the minimum version required is `5.5.9`.

I have removed the dependency and updated the pull request.

fxcoudert commented:
Thanks @JoyceBabu for the pull request!
